css-view-transitions-2/Overview.bs

```diff
@@ -453,11 +453,19 @@ The {{CSSViewTransitionRule}} represents a ''@view-transition'' rule.
 	<div algorithm>
 		To get the <dfn>resolve @view-transition rule</dfn> for a {{Document}} |document|:
 
-		1. Let |matchingRule| be the last ''@view-transition'' rule in |document| which has a ''@view-transition/navigation'' descriptor whose [=computed value=] is ''@view-transition/navigation/auto''.
+		1. Let |matchingRule| be the last ''@view-transition'' rule in |document|.
```

Comment:
I tripped on this for a moment, figured we need to filter rules based on which ones apply to the navigation. But right now all rules would apply since its a binary opt-in.

Reply:
I was wondering about the same thing. The only thing I'm not sure of is how "reloads" are skipped. Is that documented elsewhere in this spec?

Reply:
Right now we don't have to do any matching logic since there's just `auto` and `none`. Since `none` "always applies" (as per discussion on chat), and `auto` covers everything but reloads, they basically both always apply and we break the tie by picking the last one. In a reload it doesn't matter since there's currently no way to turn it on?

Once we break apart `auto` into more keywords and add `reload` this will need to be more involved.

> The only thing I'm not sure of is how "reloads" are skipped. Is that documented elsewhere in this spec?

[Here](https://drafts.csswg.org/css-view-transitions-2/#setting-up-the-view-transition-in-the-old-document:~:text=If%20navigationType%20is%20reload%2C%20then%20return)? Though that'll have to change once we add a `reload` type.

Reply:
Yes exactly